pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -111,6 +115,19 @@ rec {
     patches = [ rcu_patch ];
   };
 
+  dc_565 = generic rec {
+    version = "565.57.01";
+    url = "https://developer.download.nvidia.com/compute/nvidia-driver/redist/nvidia_driver/linux-x86_64/nvidia_driver-linux-x86_64-${version}-archive.tar.xz";
```

Comment:
Is there a reason you're using the cuda archive driver, rather than the official runfile?

https://docs.nvidia.com/datacenter/tesla/drivers/releases.json lists a driver for `565.57.01`